system/src/Grav/Common/Page/Markdown/Excerpts.php

```diff
@@ -159,20 +160,13 @@ static function ($carry, $item) {
             $locator = $grav['locator'];
             if ($type === 'link' && $locator->isStream($url)) {
                 $path = $locator->findResource($url, false) ?: $locator->findResource($url, false, true);
-                $url_parts['path'] = $grav['base_url_relative'] . '/' . $path;
+                $raw_url_parts['path'] = $grav['base_url_relative'] . '/' . $path;
                 unset($url_parts['stream'], $url_parts['scheme']);
             }
-
-            $excerpt['element']['attributes']['href'] = Uri::buildUrl($url_parts);
-
-            return $excerpt;
         }
 
-        // Handle paths and such.
-        $url_parts = Uri::convertUrl($this->page, $url_parts, $type);
```

Comment:
This call to `Utils::convertUrl()` is important as it's waht handles the relative pathing calculation.  It can't simply be removed.

Reply:
I will have a look at it.

Reply:
I see what you mean, I have updated to only encode whitespaces in the paths.